### ISSUE TITLE
Possible false positive for inferred type

### ISSUE THREAD

**Issue by dinoboy197** [HairyFotr/linter#24]
```
val base = scala.collection.mutable.Map[String, Any](
  "review_id" -> "abcd"
)
```

linter reports:
```Inferred type scala.collection.mutable.Map[String,Any]. (This might not be what you've intended)```

I did certainly intend to have a Map[String, Any].

**Comment by jawshooah** [comment]
The problem is that the type of `base` is still inferred, since it is not explicitly annotated.

Seems like the solution is this:

```scala
import scala.collection.mutable.{ Map => MutableMap }

val base: MutableMap[String, Any] = MutableMap(
  "review_id" -> "abcd"
)
```

**Comment by HairyFotr** [comment]
This does work, but the type is obvious above so I'll try to fix the check. I think I have a few similar examples stashed somewhere...

Can't promise when I'll find the time to do it though, sorry.

**Comment by dinoboy197** [comment]
@HairyFotr much appreciated!

**Comment by HairyFotr** [comment]
Ah, this case was already covered, but only for one type parameter. Should be fixed now.

Thanks again for reporting!
